src/api/public/apidocs-new/paths/source_project_name_cmd_release.yaml

```
@@ -0,0 +1,47 @@
+post:
+  summary: Release the project
```

```suggestion
  summary: Release the project.
```
Nitpick: missing dot at the end of the sentence.

Also missing in the `description` fields of the query parameters.